instrumentation/opentelemetry-instrumentation-grpc/tests/test_client_interceptor.py

```diff
@@ -187,3 +196,37 @@ def test_error_stream_stream(self):
         self.assertIs(
             span.status.status_code, trace.StatusCode.ERROR,
         )
+
+    def test_context_propagation(self):  # pylint: disable=no-self-use
```

Comment:
could you add a docstring to better describe what the test does?

Looking through the PR I now understand that we actually had a bug with injecting context propagation at all, but it would be good to make the test say something to that effect (e.g. `test_context_propagation_injects_spans`)

Reply:
Updated.